src/abstractdataframe/selection.jl

```diff
@@ -302,6 +293,16 @@ function select!(df::DataFrame, cs...)
     return df
 end
 
+"""
+    transform!(df::DataFrame, inds...)
+
+Mutate `df` in place to add columns specified by `inds...` and return it.
+Equivalent to `select!(df, :, inds...)`.
+
+See [`select!`](@ref) for a detailed rules how the transformation is applied
```

Comment:
```suggestion
See [`select!`](@ref) for detailed rules regarding accepted values for `inds`.
```
BTW, also in `select`, `inds` isn't a good name for this now, and the `select` docstring should mention immediately that it can be either an index *or a transformation*.

Reply:
OK. I will change this.